content/publishing-yaml/distribution.md

```diff
@@ -227,3 +225,17 @@ And then export the filepath on the gradlew task
         cd android && ./gradlew appDistributionUploadRelease
 
 {{</notebox>}}
+
+### Publishing web applications to Firebase Hosting
+
+Publishing web applications to Firebase Hosting With Codemagic publishing to Firebase Hosting is a straight-forward process as the Firebase CLI is already pre-installed on our virtual machines. To get started, you will need to obtain your Firebase token. In order to do that, run `firebase login:ci` in your local terminal.
```

Comment:
Publishing web applications to Firebase Hosting with Codemagic is a straight-forward process as the Firebase CLI is already preinstalled on our virtual machines.

Start a numbered list from 1. To get started, ..